lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -405,3 +405,15 @@
     src: patches/aioeventlet.py38.patch
     dest: /usr/local/lib/python3.8/dist-packages/aioeventlet.py
   when: ansible_distribution == "Ubuntu"
+
+- name: Patch python aioeventlet lib crash ubuntu
+  patch:
+    src: patches/aioeventlet_fd_exception.patch
+    dest: /usr/local/lib/python3.8/dist-packages/aioeventlet.py
+  when: ansible_distribution == "Ubuntu"
+
+- name: Patch python aioeventlet lib crash debian
+  patch:
+    src: patches/aioeventlet_fd_exception.patch
+    dest: /usr/lib/python3/dist-packages/aioeventlet.py
+  when: ansible_distribution == "Debian"
```

Comment:
@tmdzk what magic is needed to apply this for release?

Reply:
We need to add symb-link in patch dir for https://github.com/magma/magma/blob/master/third_party/build/bin/aioeventlet_build.sh.
also update iteration for package.

Reply:
I'm a bit confused is there another patch dir other than lte/gateway/deploy/roles/magma/files/patches/ ? 
Do I just need to bump ITERATION to 2, does it build automatically in CI or?

Reply:
you need to create symbolic link in this dir: https://github.com/magma/magma/tree/master/third_party/build/patches/aioeventlet

Reply:
its manual build and upload to CI.